app/assets/javascripts/home.coffee

```diff
@@ -142,58 +142,45 @@
     'Since [our daughter’s] school doesn\'t have an art curriculum, she has not been able to explore and learn about her passion until now. She has become more confident, articulate and definitely much happier since she\'s been enrolled in Doodles.'
     'With no art classes at her school, my daughter was floundering–always sketching, but frustrated by her lack of direction... We are grateful for the excellent guidance provided by Doodles Academy, which has filled a void in our daughter’s life.'
     'The online art classes provided by Doodles Academy are invaluable to me as a homeschool mom and teacher... I also appreciate the flexibility of the program. We can adjust it to our life, schedule, and health.'
+    'I absolutely love your website and have begun to use it in my classroom quite frequently ... some of my students love the website so much that they have done some lessons on their own at home.'
   ]
   quotePersons = [
     'Kara Bloom'
     'Laura Harris'
     'Erin Scott'
+    'Julie Rains'
   ]
-  quote_number = 0
-  $('#quotes p').append quotes[quote_number]
-  $('.quoted-parent').append '<p>' + quotePersons[quote_number] + '</p>'
-  $('#dot' + quote_number + ' img').attr("src", "assets/bluedot.png");
+  quoteNumber = 0
+  lastQuote = quotes.length
+  $('#quotes p').append quotes[quoteNumber]
+  $('.quoted-parent').append '<p>' + quotePersons[quoteNumber] + '</p>'
+  $('#dot' + quoteNumber + ' img').attr("src", "assets/bluedot.png");
 
```

Comment:
I wonder if we could collapse these into a simple object, instead of having two arrays. So it would be like:
```
{
  "Bilbo Baggins": "I found the Ring.",
  "Frodo Baggins": "I must destroy the Ring.",
  "Gollum": "Give me the Ring."
}
```
...and then just loop through that? Keeping it in two arrays, and then tracking the index with a local variable seems unnecessarily complicated.